src/front-door/azext_front_door/_params.py

```diff
@@ -49,6 +49,11 @@ def load_arguments(self, _):
     with self.argument_context('network front-door', arg_group='Frontend Endpoint') as c:
         c.argument('frontend_host_name', help='Domain name of the frontend endpoint.')
 
+    with self.argument_context('network front-door', arg_group='HTTPS') as c:
+        c.argument('secret_name', help='The name of the Key Vault secret representing the full certificate PFX')
+        c.argument('secret_version', help='The version of the Key Vault secret representing the full certificate PFX')
+        c.argument('vault_id', help='The resource id of the Key Vault containing the SSL certificate')
```

Comment:
Some opportunities for QOL improvements here. If the user omits version, does it use latest? Could the user provide the URL for the secret (which can include the version, no?).  Could you use that secret's URL to figure out the KeyVault ID?  

Nothing critical or blocking, but some things that I think other command modules do to try and make working with KeyVault easier.

Reply:
Version, secret name and keyvault id are mandatory parameters. All these are in line with our portal experience and powershell. Thanks for the feedback.